### ISSUE TITLE
Add dark mode to the actix.rs/docs/

### ISSUE THREAD

**Issue by rishadbaniya** [actix/actix-website#210]
A dark mode would be very nice to have

**Comment by peter-donovan** [comment]
I took a look at the existing styles and everything looks quite organized. Adding a second theme to the site will essentially need to be done by hand. I have experience working with dark themes and am comfortable deriving a dark mode for the website.

It seems that people would like this feature (myself included).

I see that this issue has been open since February and has seen no activity. I simply want to make sure that the owner/maintainer gives the "go ahead" before starting on this issue. The implementation will take time, effort and possibly a few iterations (as design-related features typically do!) before being accepted and merged.

Please feel free to share your thoughts, requests, suggestions, etc.
